share/arno-iptables-firewall/plugins/20parasitic-net.plugin

```diff
@@ -0,0 +1,249 @@
+# ------------------------------------------------------------------------------
+#           -= Arno's iptables firewall - Parasitic Network plugin =-
+#
+PLUGIN_NAME="Parasitic Network plugin"
+PLUGIN_VERSION="1.00 BETA"
+PLUGIN_CONF_FILE="parasitic-net.conf"
+#
+# Last changed          : July 20, 2017
+# Requirements          : AIF 2.0.1+
+# Comments              : This plugin allows "clients" on the same subnet to use this
+#                       : device as a gateway upstream. This network of "clients" is
+#                       : SNAT'ed to this device's first external interface.
+#                       : 
+#                       : This Parasitic Network is useful for situations when the
+#                       : upstream firewall is not under your control and you desire
+#                       : added security for specific devices in your subnet.
+#                       : Set the gateway address of Parasitic Network clients to the
+#                       : IPv4 address of this device.
+#
+# Author                : (C) Copyright 2017 by Arno van Amersfoort
```

Comment:
Yeah but you contributed, rewrote it. So prevent a discussion and just add it ;-)